docs/en/observability/synthetics-configuration.asciidoc

```diff
@@ -52,17 +52,52 @@ An object that defines any variables your tests require.
 [[synthetics-configuration-playwright-options]]
 = `playwrightOptions`
 
-For available options, see the https://playwright.dev/docs/test-configuration[Playwright documentation].
+For all available options, refer to the https://playwright.dev/docs/test-configuration[Playwright documentation]. 
+
+Below are details on a few Playwright options that are particularly relevant to Elastic Synthetics including timeouts, timezones, and device emulation.
+
+[discrete]
+[[synthetics-configuration-playwright-options-timeouts]]
+== Timeouts
 
-[NOTE]
-====
 Playwright has two types of timeouts that are used in Elastic Synthetics:
 https://playwright.dev/docs/test-timeouts#action-and-navigation-timeouts[action and navigation timeouts].
 
 Elastic Synthetics uses a default action and navigation timeout of 50 seconds.
 You can override this default using https://playwright.dev/docs/api/class-testoptions#test-options-action-timeout[`actionTimeout`] and https://playwright.dev/docs/api/class-testoptions#test-options-navigation-timeout[`navigationTimeout`]
 in `playwrightOptions`.
-====
+
+[discrete]
+[[synthetics-configuration-playwright-options-timezones]]
+== Timezones
+
+By default monitors will run in the ??? timezone, but you can specify a timezone in
+`playwrightOptions` on a per monitor or global basis.
+
+To set the timezone for all monitors in the project, set https://playwright.dev/docs/emulation#locale--timezone[`locale` and `timezoneId`]
```

Comment:
I don't think this link is correct (it's linking to `https://playwright.dev/docs/emulation#locale%E2%80%94%E2%80%8Btimezone`)